Libplanet.Tests/Action/StaticActionLoaderTest.cs

```diff
@@ -0,0 +1,38 @@
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Action.Loader;
+using Libplanet.Tests.Common.Action;
+using Xunit;
+
+namespace Libplanet.Tests.Action
+{
+    public class StaticActionLoaderTest
```

Comment:
```suggestion
    public class SimpleActionLoader
```

`SingleActionLoader` also may be intuitive.

Reply:
Actually, it was originally named `SingleActionLoader`. 😅
I've changed it back to `SingleActionLoader`.